ibis/pandas/core.py

```diff
@@ -324,6 +381,18 @@ def main_execute(expr, params=None, scope=None, aggcontext=None, **kwargs):
     if scope is None:
         scope = {}
 
+    if timecontext is None:
+        timecontext = {}
+    else:
+        try:
+            timecontext = tuple(map(pd.to_datetime, timecontext))
+        except ValueError:
+            raise com.IbisError(
+                'Cannot resolve timecontext for expression\n{}.'.format(
+                    expr.get_name()
```

Comment:
i don't think this error message is that explanatory, add the `timecontext` as well as a `from e` to show what exactly failed

Reply:
+1, fixed texts